src/app/modules/group/components/add-group/add-group.component.ts

```diff
@@ -38,14 +39,25 @@ export class AddGroupComponent {
     },
   ];
 
+  state: 'addingGroup' | 'ready' = 'ready';
+
   constructor(
     private groupCreationService: GroupCreationService,
+    private actionFeedbackService: ActionFeedbackService,
     private router: Router,
   ) {}
 
   addChild(group: AddedContent<GroupType>): void {
-    this.groupCreationService.create(group.title, group.type).subscribe(createdId => {
-      void this.router.navigate([ 'groups', 'by-id', createdId, 'details' ]);
+    this.state = 'addingGroup';
+    this.groupCreationService.create(group.title, group.type).subscribe({
+      next: createdId => {
+        this.state = 'ready';
+        void this.router.navigate([ 'groups', 'by-id', createdId, 'details' ]);
```

Comment:
Sorry I forgot: Could you add success action feedback on success as well ? ("Group successfully created")

Reply:
Fixed